includes/user.php

```diff
@@ -237,6 +236,25 @@ public static function ajax_set_admin_notice_viewed() {
 		wp_die();
 	}
 
+	public static function set_notice_viewed( $notice_id, $is_viewed = true, $meta = null ) {
```

Comment:
```suggestion
	public static function set_user_notice_viewed( $notice_id, $is_viewed = true, $meta = null ) {
```

maybe? just to keep the same convention

maybe even:

```suggestion
	public static function set_user_notice( $notice_id, $is_viewed, $meta = null ) {
```

Reply:
Done.